### ISSUE TITLE
Bug Report: Input offset crashes Web FNF (0.3.0)

### ISSUE THREAD

**Issue by PukaCyi** [FunkinCrew/Funkin#2208]
[weed]: <> (FILL THIS ISSUE THING OUT AS MUCH AS POSSIBLE)
[weed]: <> (OR ELSE YOUR ISSUE WILL BE LESS LIKELY TO BE SOLVED!)
[weed]: <> (DO NOT POST ABOUT ISSUES FROM OTHER FNF MOD ENGINES! I CANNOT AND PROBABLY WON'T SOLVE THOSE!)
[weed]: <> (GO TO THEIR RESPECTIVE GITHUB ISSUES AND REPORT THEM THERE LOL!)

[weed]: <> (ALSO MAKE SURE THAT YOU USE PROPER LABELS, IF YOU'RE RUNNING INTO COMPILER ISSUES, USE THE compiler issue LABEL!!!)

### If you are playing the game in a browser, what site are you playing it from?

[weed]: <> (Put an X in the [ ] thingies to fill out checkbox!)
[weed]: <> (something like [x] pretty much, don't screw up or you will look stupid)

- [ X ] [Newgrounds](https://www.newgrounds.com/portal/view/770371)
- [ ] [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below
- - [ ] Windows
- - [ ] Mac
- - [ ] Linux

### If you are playing the game in a browser, what browser are you using?
[weed]: <> (Again, put an x in the [ ] box!)

- [ X ] Google Chrome (or chomium based like Brave, vivaldi, MS Edge)
- [ ] Firefox
- [ ] Safari

## What version of the game are you using? Look in the bottom left corner of the main menu. (ex: 0.2.7, 0.2.1, shit like that)
0.3.0 (PROTOTYPE)

## Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

Settings > Input Offset

## Please describe your issue. Provide extensive detail and images if possible.
Pressing Input Offset instantly crashes the game

## If you're game is FROZEN and you're playing a web version, press F12 to open up browser dev window, and go to console, and copy-paste whatever red error you're getting

Could not find animation for char "`"
2Funkin.js:213267 source/funkin/Conductor.hx:380: [CONDUCTOR] Forcing BPM to 60
Funkin.js:308695 
Called from .new haxe_ValueException (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 217548 column 17)
Called from haxe_Exception.thrown (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 215590 column 11)
Called from funkin_play_notes_notestyle_NoteStyle.applyStrumlineFrames (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 174340 column 25)
Called from funkin_play_notes_StrumlineNote.setup (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 173845 column 13)
Called from .new funkin_play_notes_StrumlineNote (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 173800 column 7)
Called from .new funkin_play_notes_Strumline (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 173137 column 15)
Called from funkin_ui_debug_latency_LatencyState.create (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 204699 column 20)
Called from funkin_ui_options_OptionsState.resetSubState (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 21724 column 19)
Called from funkin_ui_options_OptionsState.tryUpdate (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 21761 column 9)
Called from flixel_FlxGame.update (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 20939 column 15)
Funkin.js:308697 Could not load spritesheet for note style: funkin
Funkin.js:308705 
Called from <unknown>.__handleError (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 308704 column 55)
Called from <unknown>.__broadcastEvent (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 308486 column 13)
Called from <unknown>.__onLimeRender (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 309576 column 8)
Called from <unknown>.dispatch (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 289248 column 16)
Called from <unknown>.handleApplicationEvent (https://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r99.zip/Funkin.js line 356 column 23)
Funkin.js:308710  Uncaught Could not load spritesheet for note style: funkin

**Comment by ThatOneCalculator** [comment]
Can reproduce on Linux Chromium

**Comment by SilverT4** [comment]
https://github.com/FunkinCrew/Funkin/assets/41199354/d3c3bedf-1373-46fb-9a7e-148516ccb3c7

Can reproduce in Chromium 126.0.6451.0 on Windows.

**Comment by PukaCyi** [comment]
dear god (first 0.3.0 bug report) and the wave of bug reports after this one is insane 

back to topic: this appears to just be a HTML5 thing I guess.

**Comment by ThatOneCalculator** [comment]
> this appears to just be a HTML5 thing I guess

That doesn't make any sense. How could it possibly be an issue with the browser's markup renderer when it's very clearly a JavaScript error in `Funkin.js`?

```
 Funkin.js:308697 Could not load spritesheet for note style: funkin
```

**Comment by PukaCyi** [comment]
Meant to refer to Funkin-Web build (which I call html5 build instead though I don't know if I was using the proper terminology or not) - didn't intend to refer to the browser being the issue

**Comment by PukaCyi** [comment]
> > this appears to just be a HTML5 thing I guess
> 
> That doesn't make any sense. How could it possibly be an issue with the browser's markup renderer when it's very clearly a JavaScript error in `Funkin.js`?
> 
> ```
>  Funkin.js:308697 Could not load spritesheet for note style: funkin
> ```
> 
> My best guess would be that it's a resource fetch error, but it can't be properly debugged due to the 0.3.0 build currently being closed-source.

Meant to refer to Funkin-Web build (which I call html5 build instead though I don't know if I was using the proper terminology or not) - didn't intend to refer to the browser being the issue

**Comment by GamerGuest** [comment]
How do i fix the *Could not load spritesheet for note style*?

**Comment by Niox124** [comment]
https://github.com/FunkinCrew/Funkin/assets/173632089/3b67363f-b1cc-4bfb-825e-f59e91c73694

it cash evry time and i cant play it

**Comment by ThatOneCalculator** [comment]
@Niox124 that has nothing to do with this issue. Please file a new issue.

**Comment by Niox124** [comment]
@ThatOneCalculator  done

**Comment by Niox124** [comment]
and it got deleted what the hell?

**Comment by EliteMasterEric** [comment]
Why is this STILL a valid issue

**Comment by EliteMasterEric** [comment]
Should be fixed in v0.5.1
